compose/config/config.py

```diff
@@ -149,6 +162,35 @@ def find(base_dir, filenames):
         [ConfigFile.from_filename(f) for f in filenames])
 
 
+def get_config_version(config_details):
+    def get_version(config):
+        if config.config is None:
+            return None
+        version = config.config.get('version', 1)
+        if isinstance(version, dict):
+            version = 1
+        return version
+
+    main_file = config_details.config_files[0]
+    validate_top_level_object(main_file)
+    version = get_version(main_file)
+    for next_file in config_details.config_files[1:]:
+        validate_top_level_object(next_file)
+        next_file_version = get_version(next_file)
+        if version is None:
+            version = next_file_version
+            continue
```

Comment:
Can't we just assume the version is 1? An "empty" v2 file should look like this, in my opinion:

```yaml
version: 2
```